React/Views/RCTTextField.m

```diff
@@ -42,6 +42,25 @@ - (void)setText:(NSString *)text
   }
 }
 
+- (void)_setupPlaceholder {
```

Comment:
@brentvatne i tested it. Working fine. I think its related with pointer on a `placeholder` property, i testing it. i setup `null` dynamicaly on placeholder from JS then setup to new value and all changes working as I expected. I testing on `TestInputExample.js` file with next code:

```javascript
<View>
    <TextInput autoCapitalize="none" placeholder={this.state.curText=="null" ?null:this.state.curText} 
    autoCorrect={false} 
    onFocus={()=> this.updateText('onFocus')} onBlur={() => this.updateText('onBlur')} 
    onChange={(event) => this.updateText( event.nativeEvent.text )} 
    onEndEditing={(event) => this.updateText( 'onEndEditing text: ' + event.nativeEvent.text )} 
    onSubmitEditing={(event) => this.updateText( 'onSubmitEditing text: ' + event.nativeEvent.text )} 
    style={styles.default} />
        
        <TextInput style={styles.default} onChange={(event)=> this.updateText( event.nativeEvent.text )}/>
            <Text style={styles.eventLabel}>
                {this.state.curText}{'\n'} (prev: {this.state.prevText})
            </Text>
</View>
```